org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/javaeditor/codemining/CalleeJavaMethodParameterVisitor.java

```diff
@@ -154,6 +163,18 @@ private boolean skipParameterNameCodeMining(IMethod method, List<?> arguments, i
 		}
 	}
 
+	private boolean skipParameterNameCodeMining(IMethodBinding mbinding, List<?> arguments, int parameterIndex) {
```

Comment:
This method is a duplicate of the other `skipParameterNameCodeMining(...)` method, except for how we get the `parameterNames` array. Please remove the code duplication.